website/common/script/fns/randomDrop.js

```diff
@@ -91,8 +97,14 @@ export default function randomDrop (user, options, req = {}, analytics) {
     } else if (rarity > 0.3) { // eggs 30% chance
       drop = cloneDropItem(randomVal(content.dropEggs));
 
-      user.items.eggs[drop.key] = user.items.eggs[drop.key] || 0;
-      user.items.eggs[drop.key] += 1;
+      user.items.eggs = {
```

Comment:
only the second one here is necessary to trigger Vue reactivity

Reply:
see previous comment